.travis.yml

```diff
@@ -21,6 +25,6 @@ script:
 - if test x"$TOKENV" != x'py24' ; then  tox ; fi
 - if test x"$TOKENV" = x'py24' ; then python2.4 -V && python2.4 -m compileall -fq -x 'module_utils/(a10|rax|openstack|ec2|gce).py' lib/ansible/module_utils ; fi
```

Comment:
Ok, this works because of a similar misspelling of `TOKENV=py24` at the top, while `TOXENV=py26` and newer vars are spelled correctly.